mip/entities.py

```diff
@@ -110,6 +114,7 @@ def __radd__(
     def __iadd__(
         self, other: Union["mip.Var", "mip.LinExpr", numbers.Real],
     ) -> "mip.LinExpr":
+        raise DeprecationWarning("Inplace operations are deprecated")
```

Comment:
since all in-place ops now raise an exception it might be best to remove the code following the exception (it's unreachable).

Reply:
you're absolutely right, I wondered if you wanted to replace the exception with a warning in the logs for a couple of versions.
If we're ok with this I'll just remove the code right away